speech/api/SpeechTest/SpeechTest.cs

```diff
@@ -138,6 +138,14 @@ public void TestSyncModelSelection()
             Assert.Equal(0, output.ExitCode);
             Assert.Contains("Brooklyn", output.Stdout);
         }
+
+        [Fact]
+        public void TestSyncEnhancedModel()
+        {
+            var output = Run("sync", "-e", _audioWavPath);
+            Assert.Equal(0, output.ExitCode);
+            Assert.Contains("regular Chromecast is fine", output.Stdout);

```

Comment:
This is very specific. I would only do "Chromecast".

Or, actually, the canonical is "Chrome" – https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/speech/cloud-client/transcribe_enhanced_model_test.py

Reply:
Done